scripts/build_mod_ic.py

```diff
@@ -129,6 +129,23 @@
         },
     },
 
+    # TSSOP-16 from JEDEC MO-153AC
```

Comment:
MO-153AC is specifically the 4.40mm wide, 0.65mm lead pitch, *20 pin* TSSOP-20. I think you want MO-153AB. The reference is freely available [here](https://www.jedec.org/standards-documents/docs/mo-153-f).

The IPC-7351B code is a standard form; `SOP` here for small outline package, `65P` for 0.65mm pitch, `500X110` for 5mm long by 1.1mm tall, `-16` for 16 pins (including any pads on the underside), and `N` for nominal spacing (vs M for most or L for least). So overall `SOP65P500X110-16N`.

Please update these two issues, the rest all looks good to me.